resolwe_bio/processes/workflows/chemut.yml

```diff
@@ -22,15 +22,86 @@
         - label: INDEL_CHR2
           value: indel_chr2
       default: snv
+    - name: genome
+      label: Reference genome
+      type: data:genome:fasta
     - name: parental_strains
       label: Parental strains
       type: list:data:alignment:bam
     - name: mutant_strains
       label: Mutant strains
       type: list:data:alignment:bam
-    - name: genome
-      label: Reference genome
-      type: data:genome:fasta
+    - name: Vc
+      label: Variant Calling
+      group:
+        - name: advanced
+          label: Advanced options
+          type: basic:boolean
+          default: false
+          required: false
+        - name: br_and_ind_ra
+          label: Do variant base recalibration and indel realignment
+          type: basic:boolean
+          default: false
+          required: false
+          hidden: "Vc.advanced === false"
+        - name: dbsnp
+          label: Use dbSNP file
+          type: basic:boolean
+          default: false
+          required: false
+          hidden: "Vc.advanced === false"
+          description: >
+            rsIDs from this file are used to populate the ID column of the output. Also, the DB INFO flag will be set when appropriate. dbSNP is not used in any way for the calculations themselves.
+        - name: known_sites
+          label: Known sites (dbSNP)
+          type: data:variants:vcf
+          required: false
+          hidden: "Vc.advanced === false || Vc.br_and_ind_ra === false && Vc.dbsnp === false"
+        - name: known_indels
+          label: Known indels
+          type: list:data:variants:vcf
+          default: []
```

Comment:
Remove this. The default cannot be set for a `data` type of inputs

Reply:
When I remove that value, the variant calling process fails:
```=======================================
Debugging information for data object 6
=======================================

Process' errors:
----------------
Internal error: 'NoneType' object is not iterable```
The issue seems to be `known_indels` evaluating to `True` [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/variant_calling/vc_chemut.yml#L235) as a result of [this change](https://github.com/genialis/resolwe-bio/pull/497/files#diff-1407ec7872daee7107b79174d3a7b37cR117). 

How can I prevent this?

Reply:
Ok, since this field is not required, it seems that the only way to overcome this issue is to include the `default: []`